packages/cli/src/commands/deploy.ts

```diff
@@ -181,7 +194,19 @@ export default class Deploy extends Command {
         task: async (ctx, task) => {
           const deployment = await deploy.runner(this.runnersToDeploy[hash], userId, projectId)
           deployment.on('data', x => task.output = x.message)
-          const runner = await deployment.promise()
+          await deployment.promise()
+        }
+      })))
+    },
+    {
+      title: 'Creating webhooks',
+      skip: () => Object.keys(this.webhooksToDeploy).length === 0,
+      task: () => new Listr(Object.keys(this.webhooksToDeploy).map(webhookKey => ({
+        title: webhookKey,
+        task: async (ctx, task) => {
+          const deployment = await deploy.webhook({ webhookKey }, userId, projectId)
```

Comment:
To keep it consistent and also this way we have the webhookHash in the db
```suggestion
          const deployment = await deploy.webhook(this.webhooksToDeploy[webhookKey], userId, projectId)
```

Reply:
the hash is already calculated on the cloud function and saved ;)